genie-web/src/main/java/com/netflix/genie/web/jpa/entities/projections/v4/IsV4JobProjection.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+package com.netflix.genie.web.jpa.entities.projections.v4;
+
+/**
+ * Projection for capturing v4 field from the database.
+ *
+ * @author standon
+ * @since 4.0.0
+ */
+public interface IsV4JobProjection {
+    /**
+     * Get the v4 flag.
```

Comment:
Nitpick but this should really be something like "Get whether this job was launched using a v4 mechanism or not" and then the return should say `true if the job is v4, false it if is v3`

We might also want to flag the whole method as `@deprecated` to make it clear we don't plan to keep it around

Reply:
done